locales/index.d.ts

```diff
@@ -500,6 +500,24 @@ export interface Locale extends ILocale {
      * リアクション
      */
     "reactions": string;
+    /**
+     * このリアクションをつける権限がありません。
+     */
+    "reactionRejected": string;
+    "_reactionRejectedReason": {
+        /**
+         * 投稿者がセンシティブなリアクションを許可していないため、リアクションできません。
+         */
+        "isSensitive": string;
+        /**
+         * この絵文字はリモートから見られないように設定されているため、リアクションできません。
+         */
+        "localOnly": string;
+        /**
+         * この絵文字をリアクションとして使うにはロールが必要です。
+         */
+        "roleIdsThatCanBeUsedThisEmojiAsReaction": string;
```

Comment:
```suggestion
        "thisEmojiRequiresRoleToBeReacted": string;
```

とかのほうが意図が明確+短いのではと思ったり

Reply:
一応、`EmojiDetailed`の型が
```ts
{
    id: string;
    aliases: string[];
    name: string;
    category: string | null;
    host: string | null;
    url: string;
    license: string | null;
    isSensitive: boolean;
    localOnly: boolean;
    roleIdsThatCanBeUsedThisEmojiAsReaction: string[];
}
```
となっていて、「どのプロパティがその絵文字を送れない原因になっているか」という情報と一致させたのですが、そうしないほうがいいですかね...？